avocado_vt/plugins/vt_joblock.py

```diff
@@ -0,0 +1,119 @@
+import errno
+import logging
+import os
+import re
+import random
+import string
+import sys
+
+from avocado.core import exit_codes
+from avocado.core.settings import settings
+from avocado.plugins.base import JobPre, JobPost
+
+from ..test import VirtTest
+
+
+class VTJobLock(JobPre, JobPost):
+
+    name = 'vt-joblock'
+    description = 'Avocado-VT Job Lock/Unlock'
+
+    def __init__(self):
+        self.log = logging.getLogger("avocado.app")
+        self.has_vt_test = False
+        self.lock_dir = settings.get_value(section="plugins.vtjoblock",
+                                           key="dir",
+                                           key_type=str,
+                                           default='/tmp')
```

Comment:
No problem, we can include the v1 and if the need comes before avocado introduces proper locking, we can resurrect this idea. I was just really surprised it didn't occur to me sooner, because it'd be way simpler. So let me re-test this...